graylog2-web-interface/src/components/cluster/index.js

```diff
@@ -0,0 +1 @@
+export GraylogClusterOverview from './GraylogClusterOverview';
```

Comment:
I can't remember that we decided anything about that. Which ES6 implementations don't support that? Anyway, I think we still have some of them around, basically I copied this one from `graylog2-web-interface/src/components/times/index.jsx`.